pcdsdevices/lasers/qmini.py

```diff
@@ -83,6 +87,47 @@ class QminiSpectrometer(Device):
     fit_stdev = Cpt(EpicsSignalRO, ':STDEV', kind='config')
     fit_chisq = Cpt(EpicsSignalRO, ':CHISQ', kind='config')
 
+    # Save spectra functions
+    def save_data(self):
```

Comment:
Nitpick, can ignore:
Is there any utility in making this more useful in non-gui contexts?
```
def save_data(self, filename: str = ""):
    if not filename:
        filename = self.file_dest.get().strip()
```

Reply:
What did you have in mind?